assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -410,7 +412,22 @@ function APIProvider({ children }) {
           status: 'publish',
         },
         method: 'POST',
-      }).then((response) => response['story_data']);
+      }).then((response) => {
+        return { ...response['story_data'], postId: response.id };
+      });
+    },
+    [customPageTemplates]
+  );
+
+  const deletePageTemplate = useCallback(
+    (id) => {
+      return apiFetch({
+        path: addQueryArgs(`${customPageTemplates}${id}/`, {
+          _method: 'DELETE',
+        }),
+        data: { force: true },
```

Comment:
This is required because reasons?

Reply:
Yes, it's the same as for media above. I'll copy the reason here, too.